### PR TITLE
Kola: add tests for Ignition v2 configs

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: kola/tests/ignition/v2/empty.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+package ignition
+
+import (
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+)
+
+func init() {
+	register.Register(&register.Test{
```
Comment (crawford) [inline kola/tests/ignition/v2/empty.go]: These will need a MinVersion, but I'm not sure what that version is going to be just yet.

**Comment by marineam** [comment]
Not particularly excited about having go-yaml nested a layer deep like that, particularly since the rest is still using path rewriting instead of the vendor/ dir magic. But it is a small thing that doesn't impact much else... ehhhh

**Comment by marineam** [comment]
LGTM though, I'll just go cry about Go dep management on my own like usual.

**Comment by crawford** [comment]
Haha, yeah.........................................................................

**Pull request merged**